src/quo2/components/tags/__tests__/status_tags_component_spec.cljs

```diff
@@ -0,0 +1,33 @@
+(ns quo2.components.tags.--tests--.status-tags-component-spec
+  (:require [quo2.components.tags.status-tags :as quo2]
+            [test-helpers.component :as h]))
+
+(defn render-status-tag
+  [opts]
+  (h/render [quo2/status-tag opts]))
+
+(h/describe "status tag component"
+  (h/test "renders status tag with positive type"
+    (render-status-tag {:status {:type :positive}
+                        :label  "Positive"
+                        :size   :small})
+    (-> (h/expect (h/get-by-test-id :status-tag-positive))
```

Comment:
As far as I know, the best practices of React Native testing library say to avoid using testID unless necessary.
Can we get by the text -  `h/get-by-text` or the translation key `h/get-by-translation-text`

Reply:
I believe for this component test we need to rely on the `testID` as the component doesn't have any translation text, and it gets the text (as the label) in via the props. The component has three types (`Positive`, `Negative`, and `Pending`) which all use the `base-tag` method/component and `testID` is the best way to identify/test all three variants.

P.S. The test suite also tests with `h/get-by-text` for the label.

Reply:
Ok but why not an accessibility label?